_docs-2-0/getting-started/design.md

```diff
@@ -112,10 +112,14 @@ called a minor compaction. A new MemTable is then created and the fact of the
 compaction is recorded in the Write-Ahead Log.
 
 When a request to read data arrives at a TabletServer, the TabletServer does a
-binary search across the MemTable as well as the in-memory indexes associated
-with each RFile to find the relevant values. If clients are performing a scan,
-several key-value pairs are returned to the client in order from the MemTable
-and the set of RFiles by performing a merge-sort as they are read.
+binary search across the MemTable as well as the index blocks associated with each RFile
+to find the relevant values. If clients are performing a scan, several key-value pairs
+are returned to the client in order from the MemTable and data blocks of RFiles by performing
+a merge-sort as they are read. If [caching] is enabled for the table, any index or data
```

Comment:
I think `sorted merge` would be more accurate than `merge-sort`.  The data is already sorted and `merge-sort` is the name of an algorithm for sorting unsorted data.

Reply:
Should be fixed now.